### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by Zejnilovic**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: 'enhancement, priority: undecided'
```
Comment (GeorgiChochov) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]:
```suggestion
labels: 'feature, priority: undecided'
```
Why enhancement?

Comment (Zejnilovic) [reply .github/ISSUE_TEMPLATE/feature_request.md:5]: It is the basic one from GitHub that is meant for features and refactoring

**Review by GeorgiChochov — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (GeorgiChochov) [reply .github/ISSUE_TEMPLATE/feature_request.md:5]: Up to you

**Review by yruslan — APPROVED** [review]
Looks cool. I should do the same for Cobrix (and Atum) as well.

**Pull request merged**